backend/app/tests/utils/json/test_json_loader.py

```diff
@@ -1,27 +1,23 @@
-import os 
 from app.utils.json import json_loader
+from app.data.wikipedia_output_path import wikipedia_output_path
```

Comment:
Or read from the .env file. (I don't recall if FastAPI supports it out of the box; if not you can use the `dotenv` library).